index.html

```diff
@@ -162,7 +338,43 @@ <h2 class="white-1 roboto-f f-weight700 l-height-24 f-size-26">I am a software d
 
     </div>
 
+    <div class="contact">
+        <p class="inter-f f-weight700 f-size-32 l-height-44 center">I am always interested in hearing about new
+            projects, so if
+            you like to chat please get in</p>
+
+        <form action="https://formspree.io/f/meqdkpzn" method="POST">
+            <label>
+                <input type="name" name="name" placeholder="Full name" maxlength="30">
+            </label>
```

Comment:
You did a great job, however, I recommend changing the `type` for the `name` from `name` to `text`.